haystack/nodes/reader/farm.py

```diff
@@ -688,6 +692,65 @@ def save(self, directory: Path):
         self.inferencer.model.save(directory)
         self.inferencer.processor.save(directory)
 
+    def save_to_remote(
+        self,
+        model_name: str,
+        hf_organization: Optional[str] = None,
+        private: Optional[bool] = None,
+        commit_message: str = "Add new model to Hugging Face.",
+    ):
+        """
+        Saves the Reader model to Hugging Face Model Hub with the given model_name. For this to work:
+        - Be logged in to Hugging Face on your machine via transformers-cli
+        - Have git lfs installed (https://packagecloud.io/github/git-lfs/install), you can test it by git lfs --version
+
+        :param model_name: Repository name of the model you want to save to Hugging Face
+        :param hf_organization: The name of the organization you want to save the model to (you must be a member of this organization)
+        :param private: Set to true to make the model repository private
+        :param commit_message: Commit message while saving to Hugging Face
+        """
+        # Note: This function was inspired by the save_to_hub function in the sentence-transformers repo (https://github.com/UKPLab/sentence-transformers/)
+        # Especially for git-lfs tracking.
+
+        token = HfFolder.get_token()
+        if token is None:
+            raise ValueError(
+                "To save this reader model to Hugging Face, make sure you login to the hub on this computer by typing `transformers-cli login`."
+            )
+
+        repo_url = create_repo(
+            token, model_name, organization=hf_organization, private=private, repo_type=None, exist_ok=True
```

Comment:
the new pylint version correctly complains about using token and model_name as positional args. Please switch to keyword args:
```suggestion
        repo_url = create_repo(
            token=token, name=model_name, organization=hf_organization, private=private, repo_type=None, exist_ok=True
```